src/components/views/context_menus/RoomTileContextMenu.js

```diff
@@ -157,9 +158,14 @@ module.exports = React.createClass({
     },
 
     _onClickForget: function() {
```

Comment:
I hate this pattern, can you plz use fat arrow functions to allow access to 'this' context. Thanks :D

Reply:
this is about the below line, if you use `() => {` on line 163 then you can do `s/self/this/`

Reply:
![image](https://user-images.githubusercontent.com/1342360/62452874-f667f280-b768-11e9-955c-5bd1f452e231.png)

:thinking:

Reply:
Not like 160, line 163 :D

Reply:
FastHub screwing up which line I tagged makes this messy, I meant to change
163 to: `MatrixClientPeg.get().forget(this.props.room.roomId).done(() => {`
and then remove 161, also replacing `self.props.room.roomId` with `this.props.room.roomId`

Reply:
Ahhhh, I see. That makes more sense and produces nicer code, good to know!